spec/system/posts_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do

```

Comment:
There are some tests failing. Please make sure all the tests pass.
![fails](https://user-images.githubusercontent.com/35504149/168151049-9851163c-54f2-40ae-a914-dc8129f04682.png)